README.md

```diff
@@ -1,41 +1,91 @@
     }
 
-### Output Format
+### Standalone
+
+i3-weather can also be invoked without using i3status to print the weather
+report in plain text.  This is mostly useful for testing your command line
+arguments without having to restart i3 every time, but you could also use it to
+check the weather from your terminal.
+
+
+    $ cd /path/to/i3-weather
+    $ .env/bin/python weather.py --api-key ...
+
+### Specifying location
+
+OpenWeatherMap's API supports 3 different ways of providing a location, which
+are supported by i3-weather as command-line arguments:
+
+  * `--zip`: look up a city by zip/postal code
+    - For countries outside the US, you must also specify `--zip-country`
+  * `--city-id`: look up a city by OpenWeatherMap's "City ID".  You can find
+    this number in the URL when searching for your location on OpenWeatherMap's
+    website (for example, Seattle is `https://openweathermap.org/city/5809844`,
+    which corresponds to `--city-id 5809844`).
+  * `--place`: look up a location by place name (typically `city,country`), e.g.
+    `Seattle,US`
+
+See [the API documentation](https://openweathermap.org/current) for more
+information about locations.
+
+### Customizing output format
 
-The script retrieves various data about the weather conditions for the provided WOEID.  You can use various format specifiers with the `--format` flag to customize the output.  The default format is `{city}, {region}: {text}, {temp}°{unit_temperature}`, which produces an output similar to `Auburn, AL: Fair, 75°F`.
+i3-weather retrieves weather information from OpenWeatherMap for the provided
+location and formats the result from a user-provided format string (given with
+`--format`).  The default format is `{city}, {country}: {text},
+{temp}°{unit_temperature}`, which produces an output similar to `Seattle, US:
+light rain, 43°F`.
 
-Your preferred unit of temperature can be set with `--unit f` or `--unit c` (this will also affect the other units: passing `--unit c` will also cause the speed unit to be `km/h` instead of `mph`)
+The default units are imperial (degrees Fahrenheit, wind speed in MPH).  You can
+change the temperature and wind speed units to metric with `--unit-temperature
+celsius` and `--unit-speed meters_sec`, respectively.
 
 The following format specifiers are supported:
 
   - `{unit_temperature}` - temperature unit (`F` or `C`)
-  - `{unit_distance}` - distance unit (`mi` or `km`)
-  - `{unit_pressure}` - pressure unit (`in` or `mb`)
-  - `{unit_speed}` - speed unit (`mph` or `km/h`)
-  - `{city}` - city associated with the input WOEID
-  - `{region}` - region (state/province) associated with the input WOEID
-  - `{country}` - country associated with the input WOEID
-  - `{wind_chill}` - windchill temperature
+  - `{unit_pressure}` - pressure unit (`hPa`)
+  - `{unit_speed}` - speed unit (`mph` or `m/s`)
```

Comment:
Did you consider just having `temp_f`/`temp_c` and `wind_speed_mph`/`wind_speed_ms` instead of requiring the units to be specified separately on the command line? That seems more in line with how the `wind_direction_*` specifiers work.

Reply:
I originally tried to make the format fully backwards compatible, but enough has to change between weather implementations that it's probably worth just doing it this way.

The reason it wasn't done this way with Yahoo is because Yahoo accepted a single units flag on the request and would include metadata in the result to tell you the units of each individual field.